.develop/serve.py

```diff
@@ -0,0 +1,61 @@
+import SimpleHTTPServer
```

Comment:
why add the python dep?

heres a popular one: https://www.npmjs.com/package/serve

Reply:
This is here so I can serve the generated doc to a local Open-RPC playground for development visibility. I was getting CORS errors with the normal `SimpleHTTPServer` on Firefox otherwise.